pkg/config/cmd/cmd.go

```diff
@@ -46,13 +55,12 @@ func (b *Bulk) Create(list *kapi.List, namespace string) []error {
 			after(info, err)
 			continue
 		}
-		data, err := info.Mapping.Codec.Encode(item)
-		if err != nil {
-			errs = append(errs, err)
-			after(info, err)
```

Comment:
@csrwng if the creation fail for the second time we should append the error to `errs` and call `after()`, right?
I think this was accidentally removed.